speech/google/cloud/speech/sample.py

```diff
@@ -43,12 +44,16 @@ class Sample(object):
                         to 16000 Hz. If that's not possible, use the
                         native sample rate of the audio source (instead of
                         re-sampling).
+
+    :type client: :class:`~google.cloud.client.Client`
+    :param client: Instance of ``Client``
```

Comment:
This is redundant. How about "the client that owns the current sample".

Also it is "(Optional) "